src/runtime/lib/API.ts

```diff
@@ -228,3 +231,18 @@ export abstract class AbstractWebView {
         );
     }
 }
+
+export abstract class AbstractTreeView {
+    protected abstract view: TreeView<PluginTreeItem<Resource>>;
+    public context: ExtensionContext;
+
+    constructor(context: ExtensionContext) {
+        this.context = context;
+    }
+
+    changeTitle(label: string): void {
```

Comment:
It is a good thing to abstract this method off. However, I don't think we need this for the treeview itself, considering we are not changing the titles for all tree views. 

Also, I wonder if by providing this abstraction layer, we are adding another interface between VSCode API and the toolkit. I think this is alright, but we can build on top of this or provide abstractions like registering an array of tree view providers etc.

Reply:
I just realize changeTitle function is never called in our codebase, thus will remove this function.